src/euphorie/client/country.py

```diff
@@ -43,6 +42,28 @@ class ClientCountry(dexterity.Container):
 
     country_type = None
 
+    # Many countries only have one language. Use it as the default
+    language_mapping_by_country = {
+        "bg": "bg",
+        "hr": "hr",
+        "cy": "el",
+        "fi": "fi",
+        "fr": "fr",
+        "de": "de",
+        "gr": "el",
+        "is": "is",
+        "it": "it",
+        "lv": "lv",
+        "lt": "lt",
+        "pt": "pt",
+        "si": "sl",
+    }
+
+    @property
+    def language(self):
+        if self.id in self.language_mapping_by_country:
+            return self.language_mapping_by_country[self.id]
```

Comment:
Isn't this equivalent to:
```
return self.language_mapping_by_country.get(self.id)
```
?

Reply:
No, far from it!
Examples: Country Cyprus (cy) has language Greek (el), Also country Greece (gr) has language Greek (el). Country Slovenia == si, has language Slovenian == sl.
Or, if we had Austria (at), it would be German (de).
There are many more examples where the country id is not the same as the language code.

Reply:
Oh, I misread your question :)
Yes, you are right.